third-party/cmake/LibvtermCMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+cmake_minimum_required(VERSION 2.8.11)
+project(libvterm LANGUAGES C)
```

Comment:
I am not familiar with all that's going on here, so take my opinion lightly. It's already uncomfortable  that we must maintain separate build files that aren't upstreamed. But then the addition of this new file introduces a pattern we aren't used to, so it's just another thing for people to grok when trying to follow the build system.